src/sasconf.h

```diff
@@ -14,7 +14,11 @@
 
 #include <sys/param.h>
 
-#define __BIGREGION__ 1
+#if defined(__BIGREGION__) && __BIGREGION__ == 0
+# undef __BIGREGION__
+#else
+# define __BIGREGION__ 1
+#endif
```

Comment:
Ooops.  You're right.
But my second point is still valid: shouldn't we set this macro automatically by default?
The person building SPHDE shouldn't need to know about SPHDE internals in order to build it...